examples/inverse/plot_use_sourcemorph.py

```diff
@@ -0,0 +1,250 @@
+"""
+================================
+Demonstrate usage of SourceMorph
+================================
+
+This example demonstrates how to morph an individual subject source estimate to
+a common reference space. It will be demonstrated using the SourceMorp class.
+The example uses parts of the MNE example
+:ref:`sphx_glr_auto_examples_inverse_plot_lcmv_beamformer_volume.py` and
+:ref:`sphx_glr_auto_examples_inverse_plot_mne_dspm_source_localization.py`.
```

Comment:
we'll need to integrate examples to document the new recommended way but let's keep it for later.